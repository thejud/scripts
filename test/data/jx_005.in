{"total":2,"items":[{"a":1},{"a":2}]}