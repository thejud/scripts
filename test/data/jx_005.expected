items
[{'a': 1}, {'a': 2}]
